changelog.txt

```diff
@@ -8,6 +8,10 @@ Legend
 + : added
 - : removed
 
+v1.0.5, 2024-??-??
```

Comment:
2024-11-27

Reply:
I usually don't bump version for minor changes. But on the other hand there are so few changes to the string map, that why not. Done.